service/controller/cluster.go

```diff
@@ -184,6 +184,18 @@ func newClusterResources(config ClusterConfig) ([]resource.Interface, error) {
 		}
 	}
 
+	var tccpfChangeDetection *changedetection.TCCPF
```

Comment:
I decided to copy/paste the TCCP detection to have a separate one that gets the logging right instead of reusing it and getting confused later when debugging the reconciliation. It would be off if you see that the TCCPF handler logs something about the TCCP stack.